src/components/appComponents/booksComponents/AddBooForm.js

```diff
@@ -23,7 +23,6 @@ function AddBookForm() {
       <h2>Add NEW BOOK</h2>
       <form onSubmit={handleSubmit}>
         <input type="text" name="title" placeholder="Book title" value={formState.title} onChange={handleChange} />
-        <input type="text" name="author" placeholder="Author name" value={formState.author} onChange={handleChange} />
         <select name="category" value={formState.category} onChange={handleChange}>
           <option value="" hidden>Category</option>
           <option value="Sci-Fi">Sci-Fi</option>
```

Comment:
- [ ] You can create a component for the form and callback the props to make it more clean.